File: PHP/card.php
<?php
require_once('payment.php');

class Card extends Payment{
     public $numberCard;
     public $dateCard;
     public $cvv;

     public function __constructor($id,$numberCard,$dateCard,$cvv){
          parent::__constructor($id);
          $this->numberCard = $numberCard;
          $this->dateCard = $dateCard;
          $this->cvv = $cvv;
     }
}


?>

File: Python/card.py
from payment import Payment

class Card(Payment):
     numberCard = int
     dateCard = str
     cvv = int

     def __init__(self, id,numberCard,dateCard,cvv):
          super.__init__(id)
          self.numberCard = numberCard
          self.dateCard = dateCard
          self.cvv = cvv
